lava-master/device-types-patch/patch-device-type.sh

```diff
@@ -9,6 +9,10 @@ cd $DEVTYPE_PATH
 for patch in $(ls /root/device-types-patch/*patch)
 do
 	echo "DEBUG: patch with $patch"
+	#cp $patch $patch.orig
```

Comment:
Why are you adding commented out lines?

Reply:
For easy debug, just comment out the line. But I can remove it.